sympy/solvers/solvers.py

```diff
@@ -899,6 +899,10 @@ def solve(f, *symbols, **flags):
         - dsolve() for solving differential equations
 
     """
+
+    if f == False:
```

Comment:
I suggest that you make this `if f is False` because solving of Relationals is supported and S.false (a value for an evaluated Relational) is equal to False (`S.false == False`) but is not the same (`S.false is False` is false). Many of the tests should not need to be modified from their original values in master (I hope).

Reply:
okay @smichr , will do